.devcontainer/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM mcr.microsoft.com/devcontainers/universal:2-linux
+
+SHELL ["/bin/bash", "-l", "-c"]
+# TODO
+# COPY ../.ruby-version /tmp/.ruby-version
+RUN RUBY_VERSION=3.1.4 && \
+    rvm install $RUBY_VERSION && \
+    rvm use $RUBY_VERSION --default && \
+    gem install dip && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
```

Comment:
I wanted to avoid this, but this is the best way to ensure this image use the correct version of ruby and always have `dip` installed.

I could also add this to our `Containerfile` as one of the options. Do you have an opinion here @klardotsh